st3/sublime_lib/path_compat.py

```diff
@@ -0,0 +1,22 @@
+__all__ = ['Path']
```

Comment:
A docstring for the motivation of this module would be appreciated.

Also maybe rename it to `_path_compat` to signal that it's supposed to private? Although users *might* potentially want to access the class type :thinking:

Reply:
I agree that the stub is suboptimal. It tries to be a pathlib.(Pure)Path but is not. We might as well just return a string if pathlib isn't available, so requiring/vendoring pathlib seems like the better approach.

I suppose until dep-deps are implemented, we won't get around vendoring to ensure proper and predictable functionality.

Reply:
Vendor'd.